packages/@aws-cdk/aws-lambda-nodejs/test/integ.function.ts

```diff
@@ -7,14 +7,23 @@ class TestStack extends Stack {
   constructor(scope: Construct, id: string, props?: StackProps) {
     super(scope, id, props);
 
+    const projectRoot = process.env.NZM_PACKAGE_SOURCE
```

Comment:
Feels odd that we will need to do that everywhere

Reply:
That's the least odd part about it. If we need to do it once we need to do it everywhere :)